libsolidity/analysis/StaticAnalyzer.h

```diff
@@ -73,8 +73,22 @@ class StaticAnalyzer: private ASTConstVisitor
 	bool visit(BinaryOperation const& _operation) override;
 	bool visit(FunctionCall const& _functionCall) override;
 
+	struct TypeComp
+	{
+		bool operator()(Type const* lhs, Type const* rhs) const
+		{
+			solAssert(lhs && rhs, "");
+			return lhs->identifier() < rhs->identifier();
```

Comment:
```suggestion
			return lhs->richIdentifier() < rhs->richIdentifier();
```
Should be the same, but faster, right?

Reply:
Once this is changed, I'll approve!